org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/utils/FilesUtilsTest.java

```diff
@@ -38,9 +38,9 @@ public void testFilesCachePathPreference() throws Exception {
 	@Test
 	public void normalizePathTest() {
 		assertEquals(Paths.get(System.getProperty("user.home"), "Test", "Folder").toString(), FilesUtils.normalizePath("~/Test/Folder"));
-		assertEquals(Paths.get(separator, "Test", "~", "Folder").toString(), FilesUtils.normalizePath("/Test/~/Folder"));
+		assertEquals(Paths.get(separator + "Test", "~", "Folder").toString(), FilesUtils.normalizePath("/Test/~/Folder"));
```

Comment:
It should belong to another PR?

Reply:
It should be, but there was a problem with git history in master. This will fix the problem.

Reply:
ok no pb